infogami/infobase/dbstore.py

```diff
@@ -440,7 +440,7 @@ def get_id(key):
                     # 'bot' column is not enabled
                     if key == 'bot' and not config.use_bot_column:
                         bots = get_bot_users(self.db)
-                        if value or str(value).lower() == "true":
+                        if value is True or str(value).lower() == "true":
```

Comment:
Please revert this change.

Reply:
flake8 actually complained about those lines that's why I touched it

Reply:
This change is OK now.